library/globals.inc.php

```diff
@@ -2613,14 +2630,15 @@ function gblTimeZones()
             xl('User Debugging Options'),
             array(
                 '0' => xl('None'),
-                '1' => xl('Display Window Errors Only'),
-                '2' => xl('Display Application Errors Only'),
-                '3' => xl('All'),
+                '1' => xl('Display Window (JavaScript) Errors Only'),
+                '2' => xl('Display Only PHP Inline Application Warnings'),
+                '3' => xl('Display All PHP Inline Logs Only'),
+                '4' => xl('Display All'),
```

Comment:
We don't want to display php warnings to display, just errors. Also, no need to log anything or try to fetch logs from environment(location issues).
I think best option here would be to leave this select as it is then add a boolean/or new selection for admin notifications. This way the user can still have their debug notification preference but admin gets their way too...

Reply:
After Googling it appears I named what that setting actually does incorrectly woops. It should've stated it only displays errors

Reply:
Ah Googling, our newest verb!